# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../glibc/glibc-2.16.0-skip-test-installation.pl.patch
# Copyright (C) 2012 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Do not run test-installation.pl script

Build fails when building in a chroot sandbox with following error
-----------------------------------------------------------------------------
CC="gcc" /usr/bin/perl scripts/test-installation.pl /TOOLCHAIN/src.glibc.mini34newtc-glibc.20120805.170542.28677.builder-urr/glibc-2.16.0/objdir-x86_64-unknown-linux-gnu/
Unmatched ( in regex; marked by <-- HERE in m/$( <-- HERE if $(abi-64-ld-soname),$(abi-64-ld-soname),ld/ at scripts/test-installation.pl line 173, <LDD> line 1.
make[1]: *** [install] Error 9
-----------------------------------------------------------------------------

Reference:

* http://www.mail-archive.com/lfs-dev@linuxfromscratch.org/msg17475.html

--- glibc-2.16.0/Makefile.orig	2012-08-05 17:23:36.910553211 +0200
+++ glibc-2.16.0/Makefile	2012-08-05 17:24:28.428672183 +0200
@@ -111,9 +111,6 @@
 				       $(slibdir) $(libdir)
 ifneq (no,$(PERL))
 ifeq (/usr,$(prefix))
-ifeq (,$(install_root))
-	CC="$(CC)" $(PERL) scripts/test-installation.pl $(common-objpfx)
-endif
 endif
 endif
 endif
